README.md

```diff
@@ -422,6 +422,7 @@ Table of Contents
   * [percy.io](https://percy.io) - Add visual testing to any web app, static site, style guide, or component library.  Unlimited team members, Demo app and unlimited projects, 5,000 snapshots / month.
   * [reflect.run](https://reflect.run) - Codeless automated tests for web apps. Tests can be scheduled in-app or executed from a CI/CD tool. Each test run includes a full video recording along with console and network logs. The free tier includes an unlimited number of saved tests, with 25 test runs per month and up to 3 users.
   * [saucelabs.com](https://saucelabs.com/) — Cross browser testing, Selenium testing and mobile testing, [free for Open Source](https://saucelabs.com/open-source)
+  * [TestCafe](https://devexpress.github.io/testcafe/) - Free and open source tool for web e2e testing with no limitations. 1 minute to setup (no WebDriver required), easy to use, cross-browser and cross-platform out-of-the box. Built-in free test parallelization. Modern JS/TS syntax. Advanced testing scenarios supported.
```

Comment:
What is the feature delivered as a service here? This looks like self hosted software to me.